addons/binding/org.openhab.binding.mihome/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="mihome"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Xiaomi Mi Smart Home Binding</name>
+    <description>This is the binding for Xiaomi Mi Smart Home.</description>
```

Comment:
Connects to Xiaomi Gateway V2 allowing to communicate with Xiaomi Mi Smart Home suite.